src/forms/inputs/checkbox-group.js

```diff
@@ -88,17 +91,45 @@ const propTypes = {
   className: PropTypes.string,
   checkboxInputProps: PropTypes.object,
   options: fieldOptionsType,
+  useDropdown: PropTypes.bool,
 }
 
 const defaultProps = {
   className: 'CheckboxGroup',
   checkboxInputProps: {},
   options: [],
+  useDropdown: false,
 }
 
-function CheckboxGroupLegend({ name, label }) {
-  if (label === false) return null
-  return <legend>{label || convertNameToLabel(name)}</legend>
+function CheckboxGroupLegend({
+  label,
+  name,
+  required,
+  requiredIndicator,
+  hint,
+}) {
+  return (
+    <legend className={classnames({ 'visually-hidden': label === false })}>
```

Comment:
source: https://www.w3.org/TR/WCAG20-TECHS/H71.html#:~:text=The%20first%20element%20inside%20the,related%20radio%20buttons%20and%20checkboxes.
> The first element inside the fieldset must be a legend element, which provides a label or description for the group.